test/e2e/create_enable_disable_delete.go

```diff
@@ -47,29 +51,27 @@ var _ = Describe("Shoot Rsyslog Relp Extension Tests", func() {
 			ctx, cancel = context.WithTimeout(parentCtx, 5*time.Minute)
 			DeferCleanup(cancel)
 
-			verifier, err := newVerifier(ctx, f.Logger, f.ShootFramework.SeedClient, f.ShootFramework.ShootSeedNamespace(), "local", f.Shoot.Name, string(f.Shoot.UID))
+			echoServerPodIf, echoServerPodName, err := common.GetEchoServerPodInterfaceAndName(ctx, f.ShootFramework.SeedClient)
```

Comment:
Why the echo-server gets deployed to the Seed for the e2e tests and to the Shoot for the testmachinery tests?

Reply:
The echo server simply gets deployed in the kind cluster for convenience when developing locally - it's easier to check for logs directly from the kind cluster instead of having to target each shoot cluster individually.

For the testmachinery tests I chose to deploy it in the shoot so as not to modify anything on the seeds - in case deletion of the echo server fails for some reason.

Reply:
Ok, thanks for explaining!